### PR TITLE
add sve40 test results for Cumulocity

### PR DESCRIPTION
**Pull request by gocadimic**

### CODE REVIEW

File: content/en/listing.md [hunk 1]
```diff
@@ -384,8 +384,20 @@ Below you can read more about each company's product and how it's utilizing LwM2
...
     <tr>
       <td>TestFest Results</td>
+        <td>
+        <a href="https://www.openmobilealliance.org/test_events/VSVE-40/SVE-40_15-21-Nov-2023_Cumulocity-Server-LwM2M-v1_0-Test-Results.pdf" target="_blank">SVE-40 - Nov 2023 - Cumulocity v1.0 Server Results</a>
+      </td>
+    </tr>
+    <tr>
+       <td></td>
+       <td>
+        <a href="https://www.openmobilealliance.org/test_events/VSVE-40/SVE-40_15-21-Nov-2023_Cumulocity-Server-LwM2M-v1_1-Test-Results.pdf" target="_blank">SVE-40 - Nov 2023 - Cumulocity v1.0 Server Results</a>
```
Comment (jpradocueva) [inline content/en/listing.md:394]: @gocadimic The link is correct, but the title is incorrect; it indicates 1.0, but the link is for v1.1.

**Review by jpradocueva — CHANGES_REQUESTED** [review]
@gocadimic there is a problem with the title related to version v1.1

File: content/en/listing.md [hunk 2]
```diff
@@ -384,8 +384,20 @@ Below you can read more about each company's product and how it's utilizing LwM2
...
+       <td></td>
+       <td>
+        <a href="https://www.openmobilealliance.org/test_events/VSVE-40/SVE-40_15-21-Nov-2023_Cumulocity-Server-LwM2M-v1_1-Test-Results.pdf" target="_blank">SVE-40 - Nov 2023 - Cumulocity v1.1 Server Results</a>
+      </td>
+    </tr>
+    <tr>
+      <td></td>
       <td>
-        <p><a href="https://www.openmobilealliance.org/test_events/TF-35/Warsaw_2017_Nov-Cumulocity-Server-Posted-Results.pdf" target="_blank">TF Warsaw, Poland - Oct/Nov 2017 - Cumulocity v1.0 Server Results</a></p>
+        <a href="https://www.openmobilealliance.org/test_events/TF-35/Warsaw_2017_Nov-Cumulocity-Server-Posted-Results.pdf" target="_blank">TF Warsaw, Poland - Oct/Nov 2017 - Cumulocity v1.1 Server Results</p>
```
Comment (jpradocueva) [inline content/en/listing.md:400]: @gocadimic This event took place in 2017. There is no need to do the update here. Check the document and you will see the test is related to 1.0. You have rectified the problem in the Server, test event 2023.

Comment (gocadimic) [reply content/en/listing.md]: @jpradocueva, I made a mistake by missing a line. Now it's fixed.

**Review by jpradocueva — APPROVED** [review]
@gocadimic ready to merge and deploy. Thanks.

**Pull request merged**